app/views/learning_hours/_form.html.erb

```diff
@@ -49,13 +49,17 @@
   </div>
   <div class="field-card mt-4">
     <h5 class="mb-3"><%= form.label :occurred_at, "Occurred On" %>:</h5>
-    <% occurred_at = learning_hour.occurred_at || Time.zone.now %>
+    <div><%= "learning_hour.occurred_at = #{learning_hour.occurred_at}" %></div>
+    <div><%= "Time.now = #{Time.zone.now}" %></div>
+    <% occurred_at = learning_hour.occurred_at || Time.now %>
```

Comment:
We should never be using `Time.now` as that refers to system time and not the application time. [Ref](https://thoughtbot.com/blog/its-about-time-zones#three-time-zones). Can you revert these changes and instead work on [freezing the time](https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html) in the spec instead? Which spec is flaky?

Reply:
@littleforest Sorry! I completely forgot about this and I can't reproduce this test failure anymore... I'm going to close this PR. 

But thanks for those `Time` examples. They are really helpful!